src/data/redux.test.jsx

```diff
@@ -708,13 +708,17 @@ describe('Data layer integration tests', () => {
       expect(state).toMatchSnapshot();
     });
 
-    it('Should call the exams service to get latest attempt data', async () => {
-      const attemptDataUrl = `${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`;
-      axiosMock.onGet(attemptDataUrl).reply(200, { attempt });
+    it('Should call the exams service to poll/refresh latest attempt data', async () => {
```

Comment:
NIT: I think the original language here is more accurate since the polling part happens outside this function and this really does just 'get the latest attempt'